scripts/generate/helpers/sanitize.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  address: expr => `and(${expr}, shr(96, not(0)))`,
```

Comment:
Perhaps it makes sense to write the mask down directly. It's arguably similar in terms of readability and may shave a couple of instructions.

```suggestion
  address: expr => `and(${expr}, 0x000000000000000000000000ffffffffffffffffffffffffffffffffffffffff)`,
```

Reply:
It increasses the code length though. Not sure which one is cheaper.

Reply:
I checked, and after optimisation both version produce the same bytecode.

Reply:
Without optimisations, the `0x000000000000000000000000ffffffffffffffffffffffffffffffffffffffff` is bigger (and more expensive to deploy), but slightly less expensive to run.